src/plugins/dashboard/public/services/dashboard_content_management/lib/find_dashboards.ts

```diff
@@ -18,6 +18,7 @@ import { DASHBOARD_CONTENT_ID } from '../../../dashboard_constants';
 
 export interface SearchDashboardsArgs {
   contentManagement: DashboardStartDependencies['contentManagement'];
+  options?: DashboardCrudTypes['SearchIn']['options'];
```

Comment:
I added search options so that the navigation embeddable dashboard search could match **only** on title and not description.